assets/src/editor/events/ticket-assignments-manager/filters/date-filters.js

```diff
@@ -14,22 +22,25 @@ const { FormColumn } = twoColumnAdminFormLayout;
  * @function
  * @param {boolean} showArchivedDates
  * @param {boolean} showExpiredDates
- * @param {Function} toggleArchivedDates
- * @param {Function} toggleExpiredDates
- * @param {boolean} showDateFilters
- * @param {boolean} showTicketFilters
+ * @param {number} dateFiltersOffset
+ * @param {Function} setFilter
  * @return {Object} rendered date filter toggles
  */
 const DateFilters = ( {
 	showArchivedDates,
 	showExpiredDates,
-	toggleArchivedDates,
-	toggleExpiredDates,
-	showDateFilters,
-	showTicketFilters,
+	dateFiltersOffset,
+	setFilter,
 } ) => {
-	const dateFiltersOffset = showDateFilters && showTicketFilters ? 2 : 7;
-	return showDateFilters ? (
+	const toggleArchivedDates = useCallback(
+		() => setFilter( showArchivedDatesAction ),
+		[]
+	);
```

Comment:
This should have `showArchivedDatesAction` and `setFilter` as dependencies

Reply:
even if those two will not change?
are the setters returned from `useReducer` not referentially equal from render to render?

Reply:
The thing is, they are outside this component and this component has no awareness of where those values are coming from.  So if you don't add the dependencies and this component just happened to get used somewhere else (or a future maintainer uses this as a guideline for copy/pasting) that could add to unexpected behaviour.  So, yes I would add the dependencies, it doesn't hurt and it prevents the _potential_ problems I outlined.

Reply:
kk fair enuf... will do...